src/v2/Components/SavedSearchAlert/Utils/extractPills.ts

```diff
@@ -31,18 +35,24 @@ export const extractPillFromAggregation = (
 ) => {
   const { paramName, paramValue } = filter
   const aggregation = aggregationForFilter(paramName, aggregations)
+  let aggregationValues = aggregation?.counts
 
-  if (aggregation) {
-    const aggregationByValue = keyBy(aggregation.counts, "value")
+  // Use hardcoded medium values for some grids (e.g. fair, collect grids)
+  if (paramName === "additionalGeneIDs" && !aggregation) {
+    aggregationValues = hardcodedMediums as Aggregation["counts"]
+  }
```

Comment:
You can read more about this moment [here](https://www.notion.so/artsy/Spike-Add-saved-search-alerts-to-other-artwork-grids-a57a04ac02654861915f6f7d3f2edb4a#951a63b67d384ecb84dcb8895ede1e9e)